generate_params.py

```diff
@@ -841,11 +841,170 @@ def main():
             gcs_bucket='sequin-public',
             restore_transformer_dir=
             'gs://sequin-public/transformer_models/large_bert/',
-            random_keys=[i],
+            model_random_keys=[i],
             save_model=True,
             save_model_dir=os.path.join('pfam_experiment_optimizers',
                                         'large_7_' + str(i)))
 
+
+    # 10000 train families save best models
+    for i in range(7):
```

Comment:
what's the 7 from? Add comment or make a CONSTANT

Reply:
the 7 is to get 35 total jobs for the 40 job limit